generators/app/templates/userResolver.test.ts

```diff
@@ -49,13 +48,15 @@ test('test signup resolver', async () => {
   tester.test(true, mutation, {
     username: 'username',
     email: 'hudoc96@hotmail.com',
-    password: 'password'
+    password: 'password',
   });
+
   const result = await tester.graphql(mutation, {}, {}, {
-    username: 'username',
-    email: 'hudoc96@hotmail.com',
-    password: 'password'
-  });
+      username: 'username',
+      email: 'hudoc96@hotmail.com',
```

Comment:
maybe do something like `email: jan@example.com` if you don't want people knowing your email

Reply:
dont mind if its there